leshan-client-cf/src/main/java/org/eclipse/leshan/client/californium/DomainIssuerCertificateVerifier.java

```diff
@@ -73,5 +85,9 @@ public void verifyCertificate(CertificateMessage message, DTLSSession session) t
 
         // - validate server name
         validateSubject(session, receivedServerCertificate);
+
+        // TODO do we really need this ?
+        CertPath certPath = expandCertPath(messageChain, null);
```

Comment:
I'm not sure to really get the purpose of this `expandCertPath` but do we need it here ?

Reply:
Kept it in here so that it is in sync with other variants.

In Domain Issuer we just verify end-entity certificate -> if you just return messageChain and all the tests still pass then it should be fine (they should pass).

Reply:
> if you just return messageChain and all the tests still pass then it should be fine (they should pass).

I do it and tests pass as expected.

No so surprising, I just saw that `expandCertPath` starts with : 
```java 
      if (trustedCertificates == null)
            return certPath;
```
I prefer to remove it.